Makefile

```diff
@@ -0,0 +1,31 @@
+include linting.mk
+
+export GO111MODULE=on
```

Comment:
this has been default for a long time, perhaps we can omit it.

Reply:
removed!